.github/workflows/ci-pr.yml

```diff
@@ -3,6 +3,9 @@ name: CI PRs
 on:
   pull_request:
 
+env:
+  ENABLE_SECURITY_SCAN: 'false'
```

Comment:
I don't think there is a way to set an ENV on a PR is there?

Reply:
I think we can use a repo variable to set the default value for Trivy Scans.